### DISCUSSION TITLE
Scroll in chat.html

### DISCUSSION THREAD

**Discussion by breftejk** [dropways/deskapp#35] [Ideas]
Scroll in chat.html should start at the bottom, not the top (not in the contacts list)
**By the way, I would like to thank you for such an epic project!**
![image](https://user-images.githubusercontent.com/62174194/102803592-c00c3100-43b8-11eb-847c-95a81cc6e842.png)
![image](https://user-images.githubusercontent.com/62174194/102803634-d31f0100-43b8-11eb-8a10-298aedb88aa2.png)

**Comment by dropways** [comment]
Hello @breftejk 

Here is the solution for scroll start to bottom

http://manos.malihu.gr/jquery-custom-content-scroller/4/#faq-12

I have implement in next commit.

Thank you
